src/components/Header/NavBar/Authorization/Authorization.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { NavLink, withRouter } from 'react-router-dom';
+
+import { logOut } from '../../../../actions';
+
+export class Authorization extends Component {
+    rendererLogin = () => {
```

Comment:
eslint  should throw error due to the deeper indentation

Reply:
It didn't show, If I move one line then he shows me the error, I correct this.